README.md

```diff
@@ -1,28 +1,54 @@
-    3. Run the stored procedure `uspGenerate`
+1. Follow the instructions for setting up the NTBS application in the [ntbs_Beta repository](https://github.com/publichealthengland/ntbs_Beta/blob/master/ntbs-service/README.md).
+1. Follow the instructions for setting up the NTBS migration database in the [ntbs-data-migration repository](https://github.com/publichealthengland/ntbs-data-migration/blob/master/README.md).
+1. Restore a backup of the geography database by carrying out the following steps:
+    i. Connect to the `ntbs-ops-dbs.uksouth.cloudapp.azure.com\NTBS` database server in SSMS.
+    i. In the `Object Explorer` panel, right-click on the `int-geography` database and select `Tasks` -> `Back Up...`.
+    i. Click `OK`.
+    i. After some time, you should see a message informing you that the back up has been successful.
+    i. Connect to the `ntbs-ops-dbs.uksouth.cloudapp.azure.com` VM via Remote Desktop (the username and password can be found in the `ntbs-ops-dbs-credentials` secret in Azure) and locate the back up file you just created.
+    i. Zip the file.
+    i. Copy the file to a temporary location on your development machine.
+    i. Unzip the file.
+    i. In SSMS, connect to your SQL Server instance.
+    i. In the `Object Explorer` panel, right-click on `Databases` and select `Restore Database...`.
+    i. Select the `Device` radio button, and add the relevant back up file via the `...` button.
+    i. Change the name of the database from `int-geography` to `geography`.
+    i. Click `OK`.
+    i. You should see a message saying that the database has restored successfully.
+1. Set up specimen matching database:
+    i. Clone the [specimen-matching repository](https://github.com/publichealthengland/ntbs-specimen-matching).
+    i. Make a copy of the `DEV-specimen-matching.publish.xml` file, named `DEV-USER-specimen-matching.publish.xml`. This file will be ignored by git.
+    i. If the instance of SQL Server that you are using is not at `localhost` then update the `Data Source` in the `TargetConnectionString`.
+    i. Double click on this config in the `Solution Explorer` panel in Visual Studio to publish the codebase. This will build the relevant views and set up the relevant tables.
+1. Clone this repository and open `VisualStudio_ntbs-reporting.sln` in Visual Studio.
+1. Make a copy of `DEV-reporting.publish.xml` and name it `DEV-USER-reporting.publish.xml`. This file will be ignored by Git.
+1. If the instance of SQL Server that you are using is not at `localhost` then update the `Data Source` in the `TargetConnectionString`.
+1. Double-click on this config in the `Solution Explorer` panel in Visual Studio to publish the codebase. This will build the relevant views and set up the relevant tables.
+1. To populate the database, do the following:
+    i. Run the stored procedure `uspSeed` in the specmen matching database.
+    i. Run the stored procedure `uspPopulateCalendarTable`.
```

Comment:
Is it obvious this is in the reporting database?

Reply:
I have included "in the reporting database" for the relevant reporting database steps